### DISCUSSION TITLE
Error: 500: failed to create user account

### DISCUSSION THREAD

**Discussion by usman-farooq-dev** [AntlerVC/firetable#909] [Support / Q&A] [answered]
what is the reason behind this

**Comment by harinij** [comment, marked as answer]
Moving the screenshot you shared here for reference below. @Usman-Faooq are you seeing this when you try to invite someone to a workspace or right away when navigating to a project. If you can share a bit about when you see this error it would be useful. 

![image](https://user-images.githubusercontent.com/307298/199368644-56973801-d165-405f-830f-b6b83292f1e6.png)
